package.json

```diff
@@ -72,6 +72,7 @@
     "bootstrap": "4.1.3",
     "react": "16.6.3",
     "react-dom": "16.6.3",
+    "react-load-script": "0.0.6",
```

Comment:
Is it actively developed ?
Repo seems to be a bit abandoned https://github.com/blueberryapps/react-load-script#readme

Reply:
but it works nice and has 30k downloads a week https://www.npmjs.com/package/react-load-script
others libs dont give have 'decorator approach' and we need if statment

Reply:
OK, got it.